binding-library/java/src/main/java/com/microsoft/azure/functions/signalr/SignalRGroupAction.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+package com.microsoft.azure.functions.signalr;
+
+/**
+ * <p>
+ * SignalR group action (used with SignalR output binding)
+ * </p>
+ *
+ * @since 1.0.0
+ */
+public class SignalRGroupAction {
+    /**
+     * User to add to or remove from group
+     */
+    public String userId = "";
```

Comment:
How is this used? Can you please add test/sample code?

Reply:
Check sample here: [https://github.com/Azure/azure-functions-signalrservice-extension/pull/49/files](https://github.com/Azure/azure-functions-signalrservice-extension/pull/49/files). Do you suggest to add sample in comments in this file?

---
In reply to: [259940432](https://github.com/Azure/azure-functions-signalrservice-extension/pull/48#discussion_r259940432) [](ancestors = 259940432)